resources/puppetlabs/puppetdb/pql/pql-grammar-experimental.ebnf

```diff
@@ -30,7 +28,7 @@ extract = <lbracket>, [<whitespace>], [extractfields], [<whitespace>], <rbracket
 <extractfields> = fieldlist;
 
 (* Filtering *)
-<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], [modifier], [<whitespace>], <rbrace>;
+<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], {modifier | pagingclause, [<whitespace>]}, [<whitespace>], <rbrace>;
```

Comment:
Oh here is something else, what happens when someone adds two modifiers/pagination parameters? Can we get a test and a guard against this possibility? Perhaps ... the trick is to force an order to stop this, but sounds like we should at least throw an error somewhere.